apps/src/music/views/SoundsPanel.tsx

```diff
@@ -158,50 +195,135 @@ const SoundsPanel: React.FunctionComponent<SoundsPanelProps> = ({
   onPreview,
 }) => {
   const folders = library.getAllowedSounds(undefined);
-  const libraryGroupPath = library.groups[0].path;
-
-  // Generate a flat list of entries to render.  We need a flat list because
-  // we will make the headers sticky.
-  const entries: {
-    type: 'folder' | 'sound';
-    folder: SoundFolder;
-    sound?: SoundData;
-  }[] = [];
-  folders.forEach(folder => {
-    entries.push({type: 'folder', folder: folder});
-    folder.sounds.forEach(sound => {
-      entries.push({type: 'sound', folder: folder, sound: sound});
+  const libraryGroupPath = library.libraryJson.path;
+
+  const [selectedFolder, setSelectedFolder] = useState<SoundFolder>(
+    library.getFolderForSoundId(currentValue) || folders[0]
+  );
+  const [mode, setMode] = useState<string>('packs');
```

Comment:
minor: create type for this? `type Mode = "packs" | "sounds"`

Reply:
Done.